pkg/intcode/intcode_test.go

```diff
@@ -66,7 +72,9 @@
 		for _, inst := range computeInstructions {
 			instructions := make([]int, len(inst))
 			copy(instructions, inst)
-			test.EqualInt(t, Compute(instructions, JumpFalse, input), resExpected)
+			inputs := make(chan int, 1)
+			inputs <- input
+			test.EqualInt(t, Compute(instructions, JumpFalse, inputs), resExpected)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 83 characters, out of limit 80](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)